robot/EDA/resources/AffiliationsPageObject.py

```diff
@@ -140,20 +133,6 @@ def Go_to_affiliations_edit_mode(self, loc):
         self.selenium.click_element(affiliations_locators["button_save_affiliation"])
         self.eda.close_toast_message()
 
-    def _expandShadowRoot(self, element):
-        shadow_root = self.selenium.driver.execute_script('return arguments[0].shadowRoot', element)
-        return shadow_root
-
-
-    def expand_shadow_root(self):
```

Comment:
please add documentation

Reply:
This was deleted prior to code review - no documentation needed since function was removed previously.